src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -193,6 +193,34 @@ public void should_attach_exception_record_to_case_by_attach_to_case_reference()
         verifyExceptionRecordAttachesToCase(null);
     }
 
+    @Test
+    public void should_attach_exception_record_to_case_by_search_case_reference()
```

Comment:
Also do we have a test case where both searchCaseReference and attachToCaseReference are set in call back ?

Reply:
Realised this is not needed